src/components/pages/variables.jsx

```diff
@@ -32,11 +59,21 @@ const VariablesPage = React.createClass({
     countryPackageVersion: PropTypes.string.isRequired,
     variables: PropTypes.arrayOf(AppPropTypes.variable).isRequired,
   },
+  getInitialState() {
+    return {
+      labelFilter: "",
+      nameFilter: ""
```

Comment:
The `filter` suffix evokes a function to me, not a query string. What about `query` or `lookup`?